src/Console/ConsoleOutput.php

```diff
@@ -301,7 +301,7 @@ public function styles($style = null, $definition = null)
      * Get/Set the output type to use. The output type how formatting tags are treated.
      *
      * @param int|null $type The output type to use. Should be one of the class constants.
-     * @return mixed Either null or the value if getting.
+     * @return int|null Either null or the value if getting.
```

Comment:
Possibly. I'm reading up on it now. I think `@return int|void` without the return statement would be fine.